src/mappings/pair.ts

```diff
@@ -5,148 +5,158 @@ import {
+  user.save()
 }
 
 export function handleLogRemoveCollateral(event: LogRemoveCollateral): void {
-  log.info("[BentoBox:LendingPair] Log Remove Collateral {} {}", [
+  log.info('[BentoBox:LendingPair] Log Remove Collateral {} {}', [
     event.params.share.toString(),
     event.params.user.toHex(),
-  ]);
-  let lendingPair = LendingPair.load(event.address.toHex());
-  lendingPair.totalCollateralShare = lendingPair.totalCollateralShare.minus(event.params.share);
+  ])
+  const lendingPair = LendingPair.load(event.address.toHex())
+  lendingPair.totalCollateralShare = lendingPair.totalCollateralShare.minus(event.params.share)
   lendingPair.save()
 
-  let user = getUserLendingPairData(event.params.user, event.address);
-  user.userCollateralShare = user.userCollateralShare.minus(event.params.share);
-  user.save();
+  const user = getUserLendingPairData(event.params.user, event.address)
+  user.userCollateralShare = user.userCollateralShare.minus(event.params.share)
+  user.save()
 }
 
 export function handleOwnershipTransferred(event: OwnershipTransferred): void {
-  log.info("[BentoBox:LendingPair] Ownership Transfered {} {}", [
+  log.info('[BentoBox:LendingPair] Ownership Transfered {} {}', [
     event.params.newOwner.toHex(),
-    event.params.previousOwner.toHex()
-  ]);
-  let lendingPair = LendingPair.load(event.address.toHex());
-  lendingPair.owner = event.params.newOwner;
+    event.params.previousOwner.toHex(),
+  ])
+  const lendingPair = LendingPair.load(event.address.toHex())
+  lendingPair.owner = event.params.newOwner
   lendingPair.save()
 }
 
 export function handleTransfer(event: Transfer): void {
-  log.info("[BentoBox:LendingPair] Transfer {} {}", [
+  log.info('[BentoBox:LendingPair] Transfer {} {}', [
     event.params._from.toHex(),
     event.params._to.toHex(),
     event.params._value.toString(),
-  ]);
-  let sender = getUserLendingPairData(event.params._from, event.address);
-  sender.balanceOf = sender.balanceOf.minus(event.params._value);
-  sender.save();
-
-  let user = getUser(event.params._to);
-  let receiver = getUserLendingPairData(event.params._to, event.address);
-  receiver.balanceOf = receiver.balanceOf.plus(event.params._value);
-  receiver.save();
+  ])
+  const sender = getUserLendingPairData(event.params._from, event.address)
+  sender.balanceOf = sender.balanceOf.minus(event.params._value)
+  sender.save()
+
+  const user = getUser(event.params._to, event.block)
+  const receiver = getUserLendingPairData(event.params._to, event.address)
+  receiver.balanceOf = receiver.balanceOf.plus(event.params._value)
+  receiver.save()
+}
+
+export function handleLogInterestRate(event: LogInterestRate): void {
+  log.info('[BentoBox:LendingPair] Log Interest Rate {}', [event.params.rate.toString()])
+  const lendingPair = LendingPair.load(event.address.toHex())
+  lendingPair.interestPerBlock = event.params.rate
+  // I assume this will change soon based on discussions
+  if (event.params.utilization) {
```

Comment:
With the current event utilization is always reported in combination, so there does not need to be a check for this param

Reply:
Ah, yes. Good catch. I missed the zero on the first event.